howto_docker.md

```diff
@@ -0,0 +1,58 @@
+# How to use Docker and Openfisca
+
+## Run OpenFisca on Docker
+
+1. Install Docker
+2. create a docker file in the directory you wish to work on.
+```
+FROM python:2-stretch 
+RUN pip install OpenFisca-country-template
```

Comment:
From the [readme](https://github.com/openfisca/country-template/blob/master/README.md#install-this-country-package-with-pip-install):

```
pip install openfisca_country_template
```